string device
string mounted
int32 size_mb
int32 used_mb
int32 available_mb

